admin/includes/header.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Treat Street Admin</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,700i&display=swap" rel="stylesheet">
+    <link href="/styleguide/styles.css" rel="stylesheet">
+    <link href="./admin_styles.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b25da52a36.js" crossorigin="anonymous"></script>
+   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    </head>
+
+    <body>
+    <div class="container-fluid">
+    <div class="wrapper bg-light">
+    <!-- Sidebar -->
+    <nav id="sidebar" class="bg-light">
+        <div class="sidebar-header text-center">
+            <img id="admin_logo" src="includes/treatstreet_logo_ver02_logo_only1_wBACKGROUND%20%281%29.png" alt="logo">
```

Comment:
Can we rename the logo file to treatstreet_logo.png, create an img folder and drop it in there and update the reference here please?

Reply:
https://consoles.herokuapp.com/admin/product_form.php
I will take care of the logo now.

Reply:
fixed